grakn-client/src/main/java/ai/grakn/client/Client.java

```diff
@@ -55,7 +57,20 @@
             return reader.lines().collect(joining("\n"));
         }
     };
-    
+
+    public static void main(String[] args) throws IOException {
+        String confPath = System.getProperty("grakn.conf");
```

Comment:
I've done it in a separate PR #1815. I'll wait for that to merge, then use it here (so don't merge this one yet!)